apps/web/src/components/auth/CreateOrganizationForm.tsx

```diff
@@ -41,6 +46,11 @@ export function CreateOrganization({}: Props) {
       const userData = decode<IJwtPayload>(token);
 
       if (userData.environmentId) {
+        if (isFromVercel) {
+          startVercelSetup();
+
+          return;
+        }
         navigate('/');
```

Comment:
I would simplify these lines into `isFromVercel ? startVercelSetup() : navigate('/')`

Reply:
I think this is fine because it is easier to read even if it makes more code.